internal/uuid/uuid.go

```diff
@@ -4,7 +4,7 @@
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
-package uuid // import "go.mongodb.org/mongo-driver/x/mongo/driver/uuid"
```

Comment:
What exactly did this import path do before (cc @matthewdale)? It forced uuid to come from our package instead of a user's installed package of the same name (e.g. Google's)? And now, in versions of Go with modules, it's unnecessary?

Reply:
Ah got it; thank you.